README.md

```diff
@@ -166,4 +173,18 @@ config :posthog, enable_error_tracking: false
 ## Multiple PostHog Projects
 
 If your app works with multiple PostHog projects, PostHog can accommodate you. For
-setup instructions, consult the [advanced configuration guide](advanced-configuration.md).
+setup instructions, consult the [advanced configuration guide](guides/advanced-configuration.md).
+
+## Developing locally
+
+To develop locally:
+
+1. Create your integration config by running the following. You can update it accordingly.
+
+```shell
+cp config/integration.example.exs config/integration.exs
+cp config/test.exs config/dev.exs
```

Comment:
hmm this will probably be quite confusing as `dev.exs` should probably be quite different from `test.exs`.